packages/pn-pa-webapp/src/pages/components/NewNotification/__test__/Attachments.test.tsx

```diff
@@ -149,8 +149,10 @@ describe('Attachments Component', () => {
     let newAttachmentBoxes = await waitFor(() => result.queryAllByTestId('attachmentBox'));
     const deleteIcon = newAttachmentBoxes[1].querySelector('[data-testid="DeleteIcon"]');
     fireEvent.click(deleteIcon!);
-    newAttachmentBoxes = await waitFor(() => result.queryAllByTestId('attachmentBox'));
-    expect(newAttachmentBoxes).toHaveLength(1);
+    await waitFor(() => {
+      newAttachmentBoxes = result.queryAllByTestId('attachmentBox');
+      expect(newAttachmentBoxes).toHaveLength(1);
```

Comment:
why the expect condition is now into the waitFor?

Reply:
Because the test was not passing, still finding 2 attachment boxes instead of 1 and probably not waiting for the deleted one to disappear.